packages/docs/components/tag.md

```diff
@@ -159,5 +159,25 @@ Use Tags as descriptors paired with a particular object. These descriptors may b
 :::
 
 ::: slot html-scss
-## HTML & CSS
+
+## Basic example
```

Comment:
Did you figure out why this isn't full width?

Reply:
nope, going to need to log an issue for this.